skimage/filters/tests/test_gaussian.py

```diff
@@ -164,3 +164,34 @@ def test_dog_invalid_sigma2():
         difference_of_gaussians(image, 3, 2)
     with pytest.raises(ValueError):
         difference_of_gaussians(image, (1, 5), (2, 4))
+
+
+@pytest.mark.parametrize("positional_count", range(1, 5))
+def test_deprecated_positional_args(positional_count):
+    """More than 1 positionl argument are deprecated."""
```

Comment:
I would remove this 'docstring' since you didn't use one in the next test -- maybe because you find the function name to be descriptive enough? And the string to match (line 177) is more informative than this docstring.